src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -53,7 +53,7 @@ export default App.extend({
   },
   onDelete() {
     // Fake a destroy for patient-flow-actions
-    if (!this.action.get('_flow')) {
```

Comment:
this however is clearly a fix and we _need_ the flow relationship somehow, but I don't yet know how we're going to get it.